packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -309,7 +305,7 @@ contract PerpetualPositionManager is FundingRatePayer, FundingRateApplier {
     /**
      * @notice Cancels a pending withdrawal request.
      */
-    function cancelWithdrawal() external notEmergencyShutdown() updateFundingRate() nonReentrant() {
```

Comment:
Why are you removing `updateFundingRateHere()`? I think it's actually ok not to update the fee/funding multipliers because this method doesn't modify sponsor positions, but curious if this was a mistake?

Reply:
Nope, this was intended. I was just matching the regular fees.